script/deploy-canary-demo

```diff
@@ -22,3 +22,9 @@ export AWS_ACCESS_KEY_ID=$GAMMA_AWS_ACCESS_KEY_ID
 export AWS_SECRET_ACCESS_KEY=$GAMMA_AWS_SECRET_ACCESS_KEY
 npm run deploy -- -b chime-sdk-demo-gamma-canary -s chime-sdk-demo-gamma-canary -t -l
 npm run deploy -- -b chime-sdk-meeting-readiness-checker-gamma-canary -s chime-sdk-meeting-readiness-checker-gamma-canary -a meetingReadinessChecker -t -l
+
+echo "Deploying to onebox stage for canary that talks to prod Chime endpoint"
```

Comment:
Should we remove the one above echo "Deploying to devo stage for canary that talks to gamma Chime endpoint"?
Also can we use the same one as in Gamma right now?

Reply:
Yes, we can safely remove line 14-18.